auction-server/src/opportunity/api.rs

```diff
@@ -378,11 +377,7 @@ pub struct OpportunityCreateV1Svm {
     /// The chain id where the opportunity will be executed.
     #[schema(example = "solana", value_type = String)]
     pub chain_id:           ChainId,
-    /// The block hash to be used for the opportunity execution.
-    #[schema(example = "DUcTi3rDyS5QEmZ4BNRBejtArmDCWaPYGfN44vBJXKL5", value_type = String)]
-    #[serde_as(as = "DisplayFromStr")]
-    pub block_hash:         Hash,
-    /// The slot where the program params were fetched from using the RPC.
+    /// The slot where the program params were fetched from using the RPC
```

Comment:
Please add period to end of all doc strings

Reply:
done